internal/update/arduino/arduino.go

```diff
@@ -82,6 +82,15 @@ func (a *ArduinoPlatformUpdater) ListUpgradablePackages(ctx context.Context, _ f
 		return nil, err
 	}
 
+	streamLibIndex, _ := commands.UpdateLibrariesIndexStreamResponseToCallbackFunction(ctx, func(curr *rpc.DownloadProgress) {
+		slog.Debug("downloading library index", "progress", curr.GetMessage())
+	})
+
+	req := &rpc.UpdateLibrariesIndexRequest{Instance: inst, UpdateIfOlderThanSecs: 0}
```

Comment:
If the default is `0`  i will omit it
```suggestion
	req := &rpc.UpdateLibrariesIndexRequest{Instance: inst}
```